src/main/java/org/sonarlint/intellij/analysis/SonarLintTask.java

```diff
@@ -53,11 +61,13 @@ public SonarLintTask(IssueProcessor processor, SonarLintJob job, boolean backgro
    * @param background Whether it should start in the foreground or background.
    */
   protected SonarLintTask(IssueProcessor processor, SonarLintJob job, boolean modal, boolean background) {
-    super(job.module().getProject(), "SonarLint Analysis", true);
+    super(job.project(), "SonarLint Analysis", true);
     this.processor = processor;
     this.job = job;
     this.modal = modal;
     this.startInBackground = background;
+    this.console = SonarLintConsole.get(job.project());
+    ;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)